update to vanilla accordion pattern

pmahnke opened the pull request:
## Done

* update to vanilla accordion pattern on downloads and brand pages

## QA

* download the branch
* `./run`
* look at [brand section](http://0.0.0.0:8011/brand/) and the [download page](http://0.0.0.0:8011/download/)

_note, there are some javascript issues on /download, but we will fix that in another PR_

## Issue / Card

* Fixes #21 

## Screenshots

![image](https://user-images.githubusercontent.com/441217/30514392-d785dbbe-9b0b-11e7-9cef-1cb9e398cf6a.png)

webteam-app commented:
Starting demo at: http://design.ubuntu.com-pr-82.run.demo.haus/

nottrobin commented on _sass/_scratch.scss:
```diff
@@ -12,105 +12,9 @@
-}
-.ui-accordion .current_page_item a {
-    color: #333;
+.p-accordion__panel > p > label {
```
If there's a problem with the accordion pattern as it is provided by Vanilla, has it been filed as an issue?

pmahnke replied:
1. the minus sign doesn't work in the [vanilla example](https://docs.vanillaframework.io/en/patterns/accordion) either; however, that is there as an example only, so I think I need help fixing it... I looked and don't easily see why it doesn't work
2. the label question, I don't think the pattern considered this application, but the fix (`display: inline`) had to be pretty specific to override vanilla, perhaps it should be BEMed, but I think there is a larger issue with the instepping you are seeing, which I do not... can we look together tomorrow?

pmahnke replied:
also, I need help getting the filters to work.  they were no longer working when I got here and are certainly not right now.

pmahnke replied:
Oh...
3. on /brand, there are no sub-pages, so the whole accordion is open... which I agree is a bit odd, but that's what Yaili asked for.

nottrobin commented on brand/canonical-logo/index.md:
```diff
@@ -6,7 +6,7 @@ body-class: "guidelines"
 <div class="p-strip">
   <div class="row">
-    <div id="loop-guidelines" class="col-10">
+    <div id="loop-guidelines" class="col-12">
```
Why? This seems like it's inconsistent with the rest of the page.

pmahnke replied:
fixed

nottrobin commented on _includes/side-nav-scopes.html:
```diff
@@ -1,10 +0,0 @@
-
```
This file is mentioned in `_layouts/post-with-banner.html` - but AFAICT that's not mentioned anywhere. If you're going to remove this one, you should presumably remove that one too.

pmahnke replied:
removed

pmahnke commented:
ok @nottrobin ready for another review:

- I removed the accordion from /download 
- made the assets static with a yaml file
- uploaded all the assets to the asset server
- removed as much leftover js and css as I could find

nottrobin commented on _includes/header.html:
```diff
@@ -1,38 +1,49 @@
+      <div class="p-navigation__logo">
+        <a class="p-navigation__link" href="/">
+          <svg xmlns="http://www.w3.org/2000/svg" height="27px" width="172px" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 172 27"><!-- Generator: Sketch 46.2 (44496) - http://www.bohemiancoding.com/sketch -->
+ <title>Artboard</title>
```
indentation
